ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -218,84 +234,47 @@ class ProfileMainScreen extends React.PureComponent<Props> {
+              subTitle={I18n.t("profile.main.privacy.description")}
               onPress={() => navigation.navigate(ROUTES.PROFILE_PRIVACY_MAIN)}
-            >
-              <Left style={styles.itemLeft}>
-                <H3>{I18n.t("profile.main.privacy.title")}</H3>
-                <Text style={styles.itemLeftText}>
-                  {I18n.t("profile.main.privacy.description")}
-                </Text>
-              </Left>
-              <Right>
-                <IconFont
-                  name="io-right"
-                  color={variables.contentPrimaryBackground}
-                />
-              </Right>
-            </ListItem>
+              isLastItem={true}
+            />
 
-            <ListItem itemDivider={true}>
-              <Text>{I18n.t("profile.main.accountSectionHeader")}</Text>
-            </ListItem>
+            <SectionHeaderComponent
+              sectionHeader={I18n.t("profile.main.accountSectionHeader")}
+            />
 
             {/* Reset PIN */}
-            <ListItem onPress={this.confirmResetAlert}>
-              <Left style={styles.itemLeft}>
-                <H3>{I18n.t("pin_login.pin.reset.button_short")}</H3>
-                <Text style={styles.itemLeftText}>
-                  {I18n.t("pin_login.pin.reset.tip_short")}
-                </Text>
-              </Left>
-              <Right>
-                <IconFont
-                  name="io-right"
-                  color={variables.contentPrimaryBackground}
-                />
-              </Right>
-            </ListItem>
+            <ListItemComponent
+              title={I18n.t("pin_login.pin.reset.button_short")}
+              subTitle={I18n.t("pin_login.pin.reset.tip_short")}
+              onPress={this.confirmResetAlert}
+            />
 
             {/* Logout/Exit */}
-            <ListItem onPress={this.onLogoutPress}>
-              <Left style={styles.itemLeft}>
-                <H3>{I18n.t("profile.main.logout")}</H3>
-                <Text style={styles.itemLeftText}>
-                  {I18n.t("profile.logout.menulabel")}
-                </Text>
-              </Left>
-              <Right>
-                <IconFont
-                  name="io-right"
-                  color={variables.contentPrimaryBackground}
-                />
-              </Right>
-            </ListItem>
+            <ListItemComponent
+              title={I18n.t("profile.main.logout")}
+              subTitle={I18n.t("profile.logout.menulabel")}
+              onPress={this.onLogoutPress}
+              isLastItem={true}
+            />
 
-            <ListItem itemDivider={true}>
-              <Text>{I18n.t("profile.main.developersSectionHeader")}</Text>
-            </ListItem>
+            <SectionHeaderComponent
+              sectionHeader={I18n.t("profile.main.developersSectionHeader")}
+            />
 
-            <ListItem>
+            <ListItem style={styles.noRightPadding}>
```

Comment:
If for every ListItem we have to add the same style:
1) we need to change the default style
2) we need to create a wrapper component that add the style for us

Reply:
Also check the "noIndent" prop.

Reply:
I tried applying noIndent props for native-base ListItem but it is ignored until the paddingLeft: 0 props is expressed.